src/advanced/network.md

```diff
@@ -32,12 +32,18 @@ Full nodes are further subdivided into 2 categories:
     _light clients_.
 
 **Light clients** represent clients in the client-server paradigm; they connect
-to full nodes to retrieve information from the blockchain they are
-interested in, and to send transactions. Exonum provides a
-[“proofs mechanism”](../glossary.md#merkle-proof),
+to full nodes for several purposes:
+
+- to [retrieve](node-management.md#explorer-api-endpoints) information they are
+  interested in from the blockchain
+- to [subscribe](node-management.md#explorer-api-sockets) to block commit events
+  and be aware of every new accepted block
+- to send transactions.
```

Comment:
If we get an information about new block committing we can not only send transactions. I'd re-write it: 
`to subscribe to events and be aware of every new block accepted.`

Reply:
I do not get your point. Light clients have 3 different purposes. Subscription to block commit and sending transactions are different purposes. I mention them separately. Besides, note that "send transactions" was mentioned even before I added subscription to events.

Reply:
I got it. It's ok. 👍🏻 I read it as one sentence because a dot is absent after `... accepted block`.